spec/policies/comment_policy_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CommentPolicy do
+  let(:user_types) { %i[learner teacher admin] }
+
+  permissions :create?, :show?, :index? do
+    let(:comment) { build_stubbed(:comment) }
+
+    it 'grants access to all user types' do
+      user_types.each do |user_type|
+        expect(described_class).to permit(build_stubbed(user_type), comment)
+      end
+    end
+  end
+
+  permissions :destroy?, :update?, :edit? do
+    let(:comment) { build_stubbed(:comment) }
+
+    it 'grants access to the author' do
+      expect(described_class).to permit(comment.user, comment)
+    end
+
+    it 'grants no access to other learners' do
```

Comment:
Here and below: We often used "does not grant..." rather than "grants no...", but I don't have a specific preference (maybe except for consistency).

Reply:
Right! I unified the spelling to use grants ...

Reply:
Alright, but that is different from all other files, isn't it?

Reply:
Ok I am using `does not grant`

Reply:
Thanks 🙂